conda_smithy/configure_feedstock.py

```diff
@@ -495,7 +494,8 @@ def prepare_matrix_for_env_vars(matrix):
                 name, value = item
                 if name in special_conda_vars:
                     name = special_conda_vars[name]
-                    value = str(value).replace('.', '')
+                    if name.upper() != 'CONDA_R':
```

Comment:
@johanneskoester I am reluctant to change this to everything as you suggested, even though I am sure it would work maybe we should leave that to another PR.